### PR TITLE
Media queries

### PR DESCRIPTION
**Pull request by rmauritsson**

### CODE REVIEW

**Comment by bolah2009** [comment]
Currently been reviewed

File: style.css [hunk 1]
```diff
@@ -1,19 +1,926 @@
...
+	z-index: 1;
+}
+
+.deals-head-title {
+	background: #f42;
+	color: #fff;
+	padding: 10.66666667px 16px;
+	padding-left: 84px;
+	margin: 0 !important;
+	font-size: 18px !important;
```
Comment (bolah2009) [inline style.css]: `SUGGESTION`

Always look for a way to use [specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) before even considering `!important`. Learn more [here](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity#The_!important_exception)

File: README.md [hunk 2]
```diff
@@ -1 +1,10 @@
...
+# Next-Web-Clone
+Responsive web design for the website https://thenextweb.com/
+
+#BreakPoints 
+- Screen sizes about 1024px
+- Screen sizes between 720p and 1024
+- Screen sizes less than 720p
+
+#live Preview
+https://raw.githack.com/rmauritsson/Next-Web-Clone/media-queries/index.html
```
Comment (bolah2009) [inline README.md]: #### `HINT`

Use markdown links instead. For example: [Live Preview](https://raw.githack.com/rmauritsson/Next-Web-Clone/media-queries/index.html) can be achieved by doing this -> `[Live Preview](https://raw.githack.com/rmauritsson/Next-Web-Clone/media-queries/index.html)`

File: README.md [hunk 3]
```diff
@@ -1 +1,10 @@
...
\ No newline at end of file
+# Next-Web-Clone
+Responsive web design for the website https://thenextweb.com/
+
+#BreakPoints 
+- Screen sizes about 1024px
+- Screen sizes between 720p and 1024
+- Screen sizes less than 720p
+
+#live Preview
```
Comment (bolah2009) [inline README.md]: #### `HINT`

Add space to achieve a markdown heading style.

```suggestion
# Live Preview
```

File: README.md [hunk 4]
```diff
@@ -1 +1,10 @@
-# Next-Web-Clone
\ No newline at end of file
+# Next-Web-Clone
+Responsive web design for the website https://thenextweb.com/
```
Comment (bolah2009) [inline README.md]: #### `HINT` 

Check the comment on using markdown links.

File: README.md [hunk 5]
```diff
@@ -1 +1,10 @@
...
+
+#BreakPoints
```
Comment (bolah2009) [inline README.md]: #### `HINT`

Add space to achieve a markdown heading style.

```suggestion
# BreakPoints 
```

**Comment by Torres-ssf** [comment]
### Pull Request Under Review By Sérgio (TSE)

**Pull request merged**